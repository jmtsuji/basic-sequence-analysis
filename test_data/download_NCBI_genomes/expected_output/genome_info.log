[ Fri May 31 22:50:47 UTC 2019 ]: Running download_NCBI_genomes.sh
[ Fri May 31 22:50:47 UTC 2019 ]: Version: v1.3.0
[ Fri May 31 22:50:47 UTC 2019 ]: Command: download_NCBI_genomes.sh -i True test_input.list test_output 2
[ Fri May 31 22:50:47 UTC 2019 ]: #### SETTINGS ####
[ Fri May 31 22:50:47 UTC 2019 ]: input_filepath: test_input.list
[ Fri May 31 22:50:47 UTC 2019 ]: output_directory: test_output
[ Fri May 31 22:50:47 UTC 2019 ]: force_override: False
[ Fri May 31 22:50:47 UTC 2019 ]: info_only: True
[ Fri May 31 22:50:47 UTC 2019 ]: ##################
[ Fri May 31 22:50:47 UTC 2019 ]: Found '5' queries to search
[ Fri May 31 22:50:47 UTC 2019 ]: Searching for 'Chlorobium ferrooxidans': Found 1 matching assemblies
[ Fri May 31 22:50:49 UTC 2019 ]: 'GCF_000168715.1' ('Chlorobium ferrooxidans DSM 13031 (green sulfur bacteria)')
[ Fri May 31 22:50:49 UTC 2019 ]: Searching for 'Chlorobium sp. KB01': Found 1 matching assemblies
[ Fri May 31 22:50:50 UTC 2019 ]: 'GCF_001975465.1' ('Chlorobium sp. KB01 (green sulfur bacteria)')
[ Fri May 31 22:50:50 UTC 2019 ]: Searching for 'GCF_000168715.1': Found 1 matching assemblies
[ Fri May 31 22:50:52 UTC 2019 ]: 'GCF_000168715.1' ('Chlorobium ferrooxidans DSM 13031 (green sulfur bacteria)')
[ Fri May 31 22:50:52 UTC 2019 ]: Searching for 'PRJNA534305': Found 2 matching assemblies
[ Fri May 31 22:50:53 UTC 2019 ]: 'GCA_005862285.1' ('Chlorobium sp. (green sulfur bacteria)')
[ Fri May 31 22:50:53 UTC 2019 ]: 'GCA_005862225.1' ('Chlorobium sp. (green sulfur bacteria)')
[ Fri May 31 22:50:53 UTC 2019 ]: Searching for 'Random text': Found 0 matching assemblies
[ Fri May 31 22:50:54 UTC 2019 ]: download_NCBI_genomes.sh: Finished.
