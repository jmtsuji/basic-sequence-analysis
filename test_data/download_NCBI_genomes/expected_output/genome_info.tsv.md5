03b1aad92dbbbe9d41b882a3ec5d46f4  expected_output/genome_info.tsv
