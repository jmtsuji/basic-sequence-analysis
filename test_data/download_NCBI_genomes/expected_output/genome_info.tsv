query	organism	species	assembly_accession	assembly_name	genbank_ftp_link
Chlorobium ferrooxidans	Chlorobium ferrooxidans DSM 13031 (green sulfur bacteria)	Chlorobium ferrooxidans	GCF_000168715.1	ASM16871v1	ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000/168/715/GCA_000168715.1_ASM16871v1
Chlorobium sp. KB01	Chlorobium sp. KB01 (green sulfur bacteria)	Chlorobium sp. KB01	GCF_001975465.1	ASM197546v1	ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/001/975/465/GCA_001975465.1_ASM197546v1
GCF_000168715.1	Chlorobium ferrooxidans DSM 13031 (green sulfur bacteria)	Chlorobium ferrooxidans	GCF_000168715.1	ASM16871v1	ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000/168/715/GCA_000168715.1_ASM16871v1
PRJNA534305	Chlorobium sp. (green sulfur bacteria)	Chlorobium sp.	GCA_005862285.1	ASM586228v1	ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/005/862/285/GCA_005862285.1_ASM586228v1
PRJNA534305	Chlorobium sp. (green sulfur bacteria)	Chlorobium sp.	GCA_005862225.1	ASM586222v1	ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/005/862/225/GCA_005862225.1_ASM586222v1
